src/main/resources/application.properties

```diff
@@ -1,6 +1,3 @@
-#jdbc properties
-spring.jpa.hibernate.ddl-auto=none
-spring.datasource.url= jdbc:postgresql://localhost:5432/student_tracker
-spring.datasource.username=postgres
-spring.datasource.password=root
-spring.datasource.driver-class-name=org.postgresql.Driver
+spring.profiles.active=test

```

Comment:
test profile should be managed under test so there should not be a need to define spring.profiles.active=test

we can define multiple profiles in spring but there is major segregation of test and actual work.

regarding test everything needs to be under test not in main.